gc-metrics.js

```diff
@@ -17,14 +17,16 @@ const gcTypes = {
 
 const noop = () => {};
 
-export default () => {
+export default client => {
+  const promClient = client || { Counter };
```

Comment:
If we support this, I'd like only do `import Counter from 'prom-client/lib/counter';` (would have to be made into a `require`) and drop the peer dependency, but print a nice warning if it's missing